tests/conftest.py

```diff
@@ -365,3 +367,21 @@ def _cleanup():
 
     request.addfinalizer(_cleanup)
     return _create_user
+
+
+@pytest.fixture(scope="session", autouse=True)
+def testsuite_properties(record_testsuite_property, pytestconfig):
+    """
+    Configures custom testsuite properties for junit xml
+    """
+    noobaa_sa_rpm_name = get_noobaa_sa_rpm_name()
+    time_now = datetime.now()
+    time_nums = time_now.strftime("%Y%m%d_%H%M%S")
+    job_name = f"noobaa-sa-{noobaa_sa_rpm_name}-{time_nums}"
```

Comment:
I don't think we should add any timestamp into the rp_launch_name - this has to be consistent - so we are able to see the history in that specific version.

Also I think we should just keep this `noobaa_sa_rpm_name` as extra attribute like `rp_noobaa_sa_rpm` and in the job name - we should just extract version like we do for ODF - in format of `X.Y` Only - this will allow us to see history for the same version over the time - not caring for specific z-stream or patch vertsion of the build.

Reply:
@petr-balogh @vavuthu Opened a new PR https://github.com/red-hat-storage/noobaa-sa-ci/pull/45